spec/lib/forms/client_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Forms::Client do
+  subject { described_class.new(query) }
+
+  let(:query) { nil }
+
+  describe '#get_all' do
```

Comment:
Are there any specific errors that the GET call can return? Wondering if those should be covered in the tests to make sure we handle them properly. Most likely you cant get a cassette for it...

Reply:
Yeah, nothing listed right now in the dev docs. There's a Lighthouse route for an API healthcheck that I'll likely need to add later though.